ui/hooks/src/use-valid-did.ts

```diff
@@ -16,11 +14,8 @@ export function useValidDid(profileId: string, enabled?: boolean) {
     if (enabled) {
       setIsLoading(true);
       resolveDid(profileId).then(res => {
-        // case when it's an eth address and we need to verify if it's a valid address separately
-        if (res && profileId.includes('eip155')) {
-          ethers.utils.isAddress(profileId.split(':').pop())
-            ? setValidDid(true)
-            : setValidDid(false);
+        if (res && 'isEthAddress' in res) {
+          res.isEthAddress ? setValidDid(true) : setValidDid(false);
```

Comment:
the DID is still valid even if the `isEthAddress` is false

Reply:
if this logic is to show the eth logo or not then please rename this

Reply:
I think I misunderstood your point earlier. Because I thought if the isEthAddress check returns false then I would display the DID not valid page, but now I see that the DID format is still valid, so we should still display the profile page but with a broken Eth logo or something.